### PR TITLE
Add developer friendly configs/tools

### PR DESCRIPTION
**Pull request by tdonohue**
This PR makes some minor enhancements to `samvera-vagrant` to make it more developer friendly (for folks who want to use vagrant for development work):

1. Preinstall PhantomJS
2. Open up Jasmine's port (8888) to make it accessible on host machine
3. Enable SSH forwarding
4. Minor preconfiguration of Ubuntu (configure system locale, turn off console beeps, etc)

As someone who uses `samvera-vagrant` for occasional development/testing, I've found each of these to be useful additions.

### CODE REVIEW

**Review by mjgiarlo — APPROVED** [review]

**Comment by mjgiarlo** [comment]
This looks good to me. I'd like to wait for @escowles to review too before we merge, if that's OK, as I'm somewhat less experienced with Vagrant. (I see you've already requested a review from him.)

**Comment by mjgiarlo** [comment]
Thank you, @tdonohue!

**Review by escowles — APPROVED** [review]
Looks good to me — thanks @tdonohue !

**Pull request merged**
